test/jdk/java/nio/channels/unixdomain/Bind.java

```diff
@@ -185,12 +189,17 @@ public static void runTests() throws IOException {
         });
         // server bind to null: should bind to a local address
         checkNormal(() -> {
-            server = ServerSocketChannel.open(StandardProtocolFamily.UNIX);
-            server.bind(null);
-            UnixDomainSocketAddress usa = (UnixDomainSocketAddress)server.getLocalAddress();
-            if (usa.getPath().toString().isEmpty())
-                throw new RuntimeException("expected non zero address length");
-            System.out.println("Null server address: " + server.getLocalAddress());
+            UnixDomainSocketAddress usa = null;
+            try {
+                server = ServerSocketChannel.open(StandardProtocolFamily.UNIX);
+                server.bind(null);
+                usa = (UnixDomainSocketAddress) server.getLocalAddress();
+                if (usa.getPath().toString().isEmpty())
+                    throw new RuntimeException("expected non zero address length");
+                System.out.println("Null server address: " + server.getLocalAddress());
+            } finally {
+                Files.deleteIfExists(usa.getPath());
```

Comment:
`usa` can be `null` here, if it never got assigned due to some exception in the prior lines, which can lead to a `NullPointerException` here.